qiskit/visualization/counts_visualization.py

```diff
@@ -287,8 +287,8 @@ def _plotting_core(
 
     if legend and len(legend) != len(data):
         raise VisualizationError(
-            "Length of legendL (%s) doesn't match "
-            "number of input executions: %s" % (len(legend), len(data))
+            "Length of legendL ({}) doesn't match "
+            "number of input executions: {}".format(len(legend), len(data))
```

Comment:
`legendL`?

Reply:
Fixed in PR #9730

Reply:
Reverted the formatting change, but kept the spelling correction.